packages/one-app-bundler/__tests__/webpack/module/webpack.client.spec.js

```diff
@@ -22,13 +22,15 @@ jest.mock('@americanexpress/one-app-dev-bundler', () => ({ BUNDLE_TYPES: { BROWS
 
 jest.mock('../../../utils/getConfigOptions', () => jest.fn(() => ({ purgecss: {} })));
 jest.spyOn(process, 'cwd').mockImplementation(() => __dirname.split(`${path.sep}__tests__`)[0]);
-
 jest.mock('../../../utils/getMetaUrl.mjs', () => () => 'metaUrlMock');
 
 jest.mock('node:url', () => ({
   fileURLToPath: jest.fn((url) => `/mock/path/for/url/${url}`),
 }));
 
+// Mock out create resolver to return a mock path for the webpack 4 pollyfill resolves.
+jest.mock('../../../webpack/createResolver.js', () => jest.fn(() => jest.fn((request) => `/mock/path/for/request/${request}`)));
+
```

Comment:
```suggestion
jest.mock('../../../webpack/createResolver.js', jest.fn(() => jest.fn((request) => `/mock/path/for/request/${request}`)));
```
does this work? It feels like too many `jest.fns` are used here

Reply:
So I would say this doesn't exactly work. What this mock is doing is essentially making a made up location for the fallbacks as in the tests without mocking that out it wont be able to resolve. 

To further explain :
CreateResolver will try to path.resolve() any path provided to it (that is a bit simplified explanation of what happens behind it) and it breaks in the test becasue it can't resolve that package in whatever location its trying to do so

Reply:
Oh i think i mis understood your question it does work 😭

Reply:
removing the extra anon function actually broke it